### PR TITLE
CRC step by step

### PR DESCRIPTION
**Pull request by amfred**
Tested with
CodeReady Containers version: 1.35.0+751824a9
OpenShift version: 4.9.5 (bundle installed at /Applications/CodeReady Containers.app/Contents/Resources/crc_hyperkit_4.9.5.crcbundle)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,16 +35,31 @@ Simply update the files under `components/(team-name)`, and open a PR with the c
...
+4) Configure the VM using the minimum supported values. You can further increase these values if your workstation can support it: `crc config set memory 16384` and `crc config set cpus 6`
+5) Create a new VM after you adjust the memory and cpu allocation: `crc delete` and confirm with a `y`.
+6) Start the OpenShift cluster: `crc start` This command will output the OpenShift web console URL as well as the developer and kubeadmin credentials when it's finished.
+7) Set up your command line: `eval $(crc oc-env)`
+8) Configure kubectl to use the CRC administrator account: `kubectl config use-context crc-admin`
+
+### Bootstrap App Studio
 Steps:
-1) Run `hack/bootstrap-cluster.sh` which will bootstrap Argo CD (using OpenShift GitOps) and setup the Argo CD `Application` CRs for each component.
-2) Retrieve the Argo CD Web UI URL using `kubectl get routes`.
```
Comment (amfred) [inline README.md:44]: The `kubectl get routes` command doesn't output the URL for me, but it is printed out at the end of the `bootstrap-cluster.sh` script.

Comment (jduimovich) [reply README.md:44]: That's true, perhaps a picture is better for folks who are already logged into the cluster in the console. 
 
Navigate to the OpenShift Console and login to ArgoCD by selecting the OpenShift Gitops menu in the applications pulldown. 
image

Comment (amfred) [reply README.md:44]: Added.

**Review by jduimovich — APPROVED** [review]
LGTM

**Pull request merged**